### DISCUSSION TITLE
Silent Uninstall

### DISCUSSION THREAD

**Discussion by yougs21** [valinet/ExplorerPatcher#2074] [Ideas]
Hello,

How to do a fully silent uninstallation of ExplorerPatcher?

OK for install : "ep_setup /S" but how to uninstall silently ?

Thanks

**Comment by Poopooracoocoo** [comment]
Tracking in https://github.com/valinet/ExplorerPatcher/discussions/2007

**Reply by yougs21** [reply]
In the 3 ways to uninstall, there is a validation window then?

**Comment by thangarajua** [comment]
anyways to remove silently?
